platforms/documentation/docs/src/snippets/dependencyManagement/customizingResolution-secondaryVariants/groovy/build.gradle

```
@@ -0,0 +1,52 @@
+// tag::lib[]
+plugins {
+    id 'java-library'
+}
+
+group = 'com.gradle'
+version = '1.2.3'
```

```suggestion
group = "com.gradle"
version = "1.2.3"
```

We should be using double quotes consistently in both Groovy and Kotlin

---

Fixed.